outputs.tf

```diff
@@ -47,3 +47,8 @@ output "resource_id" {
   value       = join("", aws_db_instance.default.*.resource_id)
   description = "The RDS Resource ID of this instance."
 }
+
+output "master_user_secret_arn" {
```

Comment:
I think exposing the entire `master_user_secret` object would be useful for users, and will simplify this logic by avoiding those `local` values:

```hcl
output "master_user_secret" {
  value = one(aws_db_instance.default[*].master_user_secret)
  ```